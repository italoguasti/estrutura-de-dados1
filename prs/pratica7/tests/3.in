50
Pedro 14 3 1980
Sophia 6 12 1933
Bernardo 7 9 1969
Davi 25 5 1955
Helena 20 3 2001
Miguel 26 6 2002
Arthur 30 8 2014
Alice 13 3 1977
Gabriel 6 11 2009
Arthur 15 6 1930
Davi 24 5 2006
Sophia 23 3 2024
Arthur 8 5 1985
Bernardo 25 6 1930
Pedro 21 7 1983
Alice 29 3 1992
Helena 27 9 1995
Miguel 1 10 2011
Arthur 12 10 1946
Helena 18 11 2009
Helena 16 8 1946
Arthur 22 3 1981
Bernardo 31 7 1991
Isabella 20 2 1982
Gabriel 22 1 1980
Arthur 31 7 1948
Sophia 3 6 1936
Arthur 22 9 1978
Helena 28 10 1995
Davi 1 5 1932
Bernardo 9 6 2022
Alice 15 1 1938
Davi 28 5 1946
Miguel 11 4 2007
Isabella 21 1 1985
Arthur 4 4 2020
Arthur 7 3 2001
Isabella 23 8 1935
Sophia 8 7 1964
Bernardo 11 4 1968
Pedro 19 7 1950
Gabriel 8 4 2011
Gabriel 25 8 1946
Bernardo 23 12 1940
Gabriel 10 12 1954
Alice 15 9 1974
Isabella 5 11 1987
Sophia 20 9 1998
Sophia 12 6 1963
Sophia 29 3 2004
