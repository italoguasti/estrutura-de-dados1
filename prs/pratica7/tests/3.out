Analise.....: Comp | Movi
- Bubble....: 1225 | 0643
- Selection.: 1225 | 0041
- Insertion.: 0692 | 0692
